release/release_files.py

```diff
@@ -39,6 +42,8 @@
   ".devcontainer/",
   "Darwin/",
   ".vscode",
```

Comment:
```suggestion
  ".vscode/",
```

I believe we should also do this one, but I don't know if it was intentional